tools/macos/docker-machine/README.md

```diff
@@ -0,0 +1,156 @@
+# Setting up OpenWhisk with Docker-machine
+
+OpenWhisk will be running on virtual machine in which Docker daemon is running.
```

Comment:
> OpenWhisk will be running on virtual machine in which Docker daemon is running.

OpenWhisk can on a Mac using a virtual machine in which Docker daemon is running.